test/controllers/ManageGroupControllerSpec.scala

```diff
@@ -131,7 +131,7 @@ class ManageGroupControllerSpec extends BaseSpec {
       groups.size() shouldBe 3
 
       // check first group
-      html.select(H2).get(0).text() shouldBe "name 1 Custom group"
+      html.select(H2).get(0).text() shouldBe "name 1 Tax service group"
```

Comment:
these shouldn't be tax groups if the expected summaries are custom right?
- something must be up with the tag UI logic